api/runnerpool/ch_placer.go

```diff
@@ -27,7 +27,9 @@ func NewCHPlacer(cfg *PlacerConfig) Placer {
 // Because we ask a runner to accept load (queuing on the LB rather than on the nodes), we don't use
 // the LB_WAIT to drive placement decisions: runners only accept work if they have the capacity for it.
 func (p *chPlacer) PlaceCall(rp RunnerPool, ctx context.Context, call RunnerCall) error {
-
+	if call.Model().Type == models.TypeAcksync {
```

Comment:
I still think this should be a new field in the config (eg. PlayerAsyncTimeout). We override the cfg here and that's violates the placer constructors. For example, for an unsuspecting developer, it would come out as a surprise where we override the original config (eg. NewNaivePlacer()) with a new one here. In OCI side, we construct these configs from env vars. (see lb main.go) We do not customize it yet, but we can anytime in case of emergencies. Because of that, if this becomes an extra field in the cfg, it's much easier to deal with from OCI side as well as OSS.

Reply:
I agree, I am going to change it even if that was done by @JadeRedworth . ATM she is super busy with something else so I'll go ahead and modify it and make sure I'll inform her about it